packages/valory/agents/trader/aea-config.yaml

```diff
@@ -187,6 +187,11 @@ models:
       redeeming_batch_size: ${int:5}
       slippage: ${float:0.01}
       redeem_margin_days: ${int:15}
+      epsilon: ${float:0.1}
+      irrelevant_tools: ${list:["openai-text-davinci-002", "openai-text-davinci-003",
```

Comment:
should the list not be empty to start with?

Reply:
I just added these here because we definitely know they are not useful to the trader agent. We can keep the list empty, but this would mean that some txs which will cost mech fees will result in invalid responses as they do not return the required format (`p_yes`, `p_no`, etc.) until the policy stops choosing them.